### PR TITLE
[MTDB-11009] Feature/sounds settings

### PR DESCRIPTION
**Pull request by PorlyBe**
initial setup... sounds was causing big lag. so i added use-sound lib to handle it

### CODE REVIEW

File: src/components/Sidebar/Settings/styled.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import styled from 'styled-components'
```
Comment (micahpatng) [inline src/components/Sidebar/Settings/styled.ts:1]: you can remove this lol

File: src/context/store.ts [hunk 2]
```diff
@@ -43,14 +46,47 @@ export const initialState: State = {
...
-    asyncActionHandlers,
+    init,
   );
+
+  useEffect(() => {
+    const persistedState = { settings: state.settings };
+    window.localStorage.setItem(storageKey, JSON.stringify(persistedState));
+  }, [state]);
+
+  /* useEffect(() => {
```
Comment (micahpatng) [inline src/context/store.ts:80]: and this

**Review by micahpatng — APPROVED** [review]
just minor fixes and it's good to go!

File: src/context/store.ts (hunk 2 above)
Comment (PorlyBe) [reply src/context/store.ts:80]: the comments in store are for debugging purposes... uncomment if you want state/action logs

**Pull request merged**